pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -295,6 +295,7 @@ func findCustomObjectByName(template api.CustomObject, objects []api.CustomObjec
 }
 
 func getEnvTemplate(cr *api.KieApp) (envTemplate api.EnvTemplate, err error) {
+	// TODO
```

Comment:
note still needed?

Reply:
no, these are letfovers, forgot to remove them.